platforms/hyperledger-indy/configuration/roles/create/helm_component/ledger_txn/tasks/identity.yaml

```diff
@@ -0,0 +1,19 @@
+#----------------------------------------------------------------------  
```

Comment:
This task should add just ONE identity assuming that the admin identity and new identity come from different organisations. The logic which chooses the admin identity (and admin organisation) and the new identity (with its organisation) should be in the setup role which would call this one.